### ISSUE TITLE
Top of Barnet 

### ISSUE THREAD

**Issue by Mano-0** [mahemenway/DAStrat#2]
Hey Everyone,
We finished another cross section in Delaware and we're having a discussion about the top of Barnet in (Dixeland 3-1), What do you think guys?
![Delaware_B map](https://user-images.githubusercontent.com/57755581/69490976-8906db80-0e97-11ea-9ea7-cf5270a83d39.png)
[Delaware_B.pdf](https://github.com/mahemenway/DAStrat/files/3883296/Delaware_B.pdf)
![Dixeland 3-1](https://user-images.githubusercontent.com/57755581/69490977-899f7200-0e97-11ea-93d4-0ea49c6d470d.png)

**Comment by mahemenway** [comment]
I agree with the placement here. @tpratherpofg @mnietoPOFG?

**Comment by tpratherpofg** [comment]
I like the top Barnett pick, is there a unit between the Barnett and Woodford that isn't picked? Or is the Barnett sitting directly on the Woodford in this area?

**Comment by tpratherpofg** [comment]
@Mano-0 @GeoAmr

**Comment by mahemenway** [comment]
@tpratherpofg Yes, we're not picking the Miss Lime for this project. I think the pick shown here accomplishes that.
